addons/godot-next/nodes/behaviors.gd

```diff
@@ -0,0 +1,173 @@
+# author: xdgamestudios
+# license: MIT
+# description: Provides a behavior system API for the nodes. 
+# todo: Refactor code in general to clean and improve performance
+# usage:
+# - Creating:
+#     behaviors = Behaviors.new() # through code
+#     Note:
```

Comment:
This note about displaying it in the Inspector is, I think, unnecessary. So is the "through code" bit.